### PR TITLE
feat: Add changelog

### PR DESCRIPTION
**Pull request by fmerian**

### CODE REVIEW

File: changelog.mdx [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+title: 'Product updates'
+description: 'What Mintlify recently launched'
+---
+
+<Tip>
+  To upgrade your CLI, run the command below.
+</Tip>
+```
+npm i -g mintlify@latest
+```
```
Comment (hahnbeelee) [inline changelog.mdx]: why is this here?

Comment (fmerian) [reply changelog.mdx]: > why is this here?

first, i wanted to give users quick access to what they want/need. on second thought, we could remove this part as we mention it when relevant.

**Comment by fmerian** [comment]
Updated version includes:

- New navigation
- Two separate sections for product updates and launch weeks

**Comment by fmerian** [comment]
Add inputs for 2024-09

**Comment by fmerian** [comment]
added inputs for 2024-09

**Review by hahnbeelee — APPROVED** [review]
